### DISCUSSION TITLE
Ignore new planes after placement

### DISCUSSION THREAD

**Discussion by StainlessReality** [google/model-viewer#3940] [Q&A]
Hello,

The problem (fictional scenario for explanation purposes):

You are considering buying a new bed want to see what it would look like in place of your existing bed. You place the new bed on the bedroom floor in your AR session, and drag it across to where the bed would actually sit. The AR session recognises a new plane (your existing bed) and places your new bed on top of your existing bed (not the idea..).

Ideal solution:
Some way to get the AR session to constrain drag-translation to the plane on which the model was placed (effectively making it an infinite plane), and ignore any new planes detected during the session.

Thank you,
Jakub

**Comment by elalish** [comment]
That should be doable, though I'd be a bit concerned about reliability: often the first plane found isn't very accurate. I'd be interested to see your results if you hack up model-viewer.
